logstash-core/locales/en.yml

```
@@ -269,6 +269,10 @@ en:
           enabled by default.
         http_host: Web API binding host
         http_port: Web API http port
+        enable-local-plugin-development: |+
+          Allow Gemfile to be manipulated directly to facilitate simpler local plugin development.
+          This is an advanced setting, intended only for use by logstash developers, and should
+          not be used in production.
```

Since this is included in command-line `--help` output that doesn't play nicely with wrapping, we should reformat to keep the individual lines short (<40char).

```suggestion
          Allow Gemfile to be manipulated directly
          to facilitate simpler local plugin
          development.
          This is an advanced setting, intended
          only for use by Logstash developers,
          and should not be used in production.
```